sections/delegate/constants.tsx

```diff
@@ -1,3 +1 @@
-export const FORM_COL_WIDTH = 382;
-
-export const TABLE_COL_WIDTH = 495;
+export const NOOP = 1;
```

Comment:
is this used? 

also i thought a no-op would be more of a function like `() => {}`

Reply:
Gotcha. Main intent was to keep the file but I've removed it from now.